static/styles/zulip.css

```diff
@@ -891,7 +891,9 @@ li.actual-dropdown-menu i {
 }
 
 #settings_content .dependent-block,
-#invite-user .dependent-block {
+#invite-user .dependent-block,
+#stream-creation .dependent-block,
+#stream_privacy_modal .dependent-block {
```

Comment:
We probably just want to rename `dependent-block` to `settings-dependent-block`, and move the class to `app_components` and have this be common CSS for those, rather than having a list of panel IDs that happen to use this... Generally all lists of IDs like this are things we want to move away from in favor of components.

Reply:
Added commit to rename it to `dependent-settings-block` and for moving CSS to `app_components.css`.

Although dependent-block is currently used for only the custom inputs of this settings, so we could have instead made only CSS changes, but `dependent-settings-block` sounds a better name anyways so I also renamed the class too.